tests/app/service/test_rest.py

```diff
@@ -3385,3 +3390,125 @@ def test_get_returned_letter_summary(admin_request, sample_service):
     assert len(response) == 2
     assert response[0] == {'returned_letter_count': 2, 'reported_at': '2019-12-11'}
     assert response[1] == {'returned_letter_count': 1, 'reported_at': '2019-12-08'}
+
+
+@freeze_time('2019-12-11 13:30')
+def test_get_returned_letter(admin_request, sample_letter_template):
+    letter_from_previous_report = create_notification_history(
+        template=sample_letter_template,
+        client_reference='letter_from_previous_report',
+        status=NOTIFICATION_RETURNED_LETTER,
+        created_at=datetime.utcnow() - timedelta(minutes=1),
+        created_by_id=sample_letter_template.service.users[0].id
+    )
+    create_returned_letter(service=sample_letter_template.service, reported_at=datetime.utcnow() - timedelta(days=3),
```

Comment:
I'm not sure what this line is doing? You can remove it and the test still passes